### ISSUE TITLE
dotnet test not working in net35

### ISSUE THREAD

**Issue by andygonzalez2010** [nunit/nunit#3790]
Steps to reproduce:
```
dotnet new -i NUnit3.DotNetNew.Template
dotnet new nunit -n CSharpNet35  --framework net35 -lang c#
msbuild /p:FrameworkPathOverride="%programfiles(x86)%\Reference Assemblies\Microsoft\Framework\.NETFramework\v3.5\Profile\Client" CSharpNet35
dotnet test CSharpNet35 --no-build
```
It runs without errors but it shows nothing in the cmd (it should show the test results).

**Comment by jnm2** [comment]
Is the problem resolved if you add `<IsTestProject>true</IsTestProject>` in the .csproj file? (https://github.com/microsoft/vstest/issues/2763)

**Comment by rprouse** [comment]
We are planning on dropping .NET 3.5 support, so i am curious. If you are using the new project file format and the new .NET tools, why are you targeting such an old version of .NET? If you are testing a library, would it be acceptable to build your test project as .NET 4.5 to test your .NET 3.5 assemblies?

**Comment by jnm2** [comment]
@rprouse I can speak to why my company uses .NET Framework 3.5 with the new project format and the new .NET tools. Some of our smaller projects are plugins to a large platform that runs on .NET Framework 3.5. There is an end to this in sight, but it will certainly be years until we can stop compiling these projects for 3.5.

> If you are testing a library, would it be acceptable to build your test project as .NET 4.5 to test your .NET 3.5 assemblies?

For us, no. We've already had a few framework differences and don't want to guess again about that in the future.

For anyone using `dotnet test`, yes. `dotnet test` can't run your .NET Framework 3.5 tests on .NET Framework 3.5. It forces them to run on .NET Framework 4.0.

**Comment by rprouse** [comment]
@jnm2 if that is the case, is staying on NUnit 3.13.x so that you have .NET 3.5 support an issue? I assume that you are not looking for new NUnit features for these projects, only bug fixes to existing features.

**Comment by jnm2** [comment]
@rprouse Right. The reality is that we're lucky that new NUnit features are still going into the net35 build, most of all nullability annotations. All the other tools we're using are old versions. We knew this day would come and I'm in full support of NUnit v4 requiring 4.x (even 4.8, which is the only other .NET Framework target at my company).

**Comment by rprouse** [comment]
@andygonzalez2010 we haven't heard back from you, so I am closing this issue. Please comment if you need more help.
